specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/examples/WorkbookGet.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "api-version": "2021-08-01",
+    "subscriptionId": "6b643656-33eb-422f-aee8-3ac145d124af",
+    "resourceGroupName": "my-resource-group",
+    "resourceName": "deadb33f-5e0d-4064-8ebb-1a4ed0313eb2"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "c0deea5e-3344-40f2-96f8-6f8e1c3b5722",
```

Comment:
>c0deea5e [](http://example.com/codeflow?start=15&length=8)

resource's id property should be the ARM resource id

/subscriptions/<subscriptionid>/etc....